CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Change Log
 
+## 3.1.1
+
+- Removed unnecessary `ResourceOperationType` protocol
```

Comment:
Forgot to update the CHANGELOG on this [PR](https://github.com/theappbusiness/TABResourceLoader/pull/28)